.github/workflows/advanced.yml

```diff
@@ -84,6 +90,11 @@ jobs:
         run: |
           .github/scripts/get-sha.sh ${{ env.BINUTILS_REPO }} ${{ env.BINUTILS_BRANCH }}
 
+      - name: Get workflow SHA
```

Comment:
Thouch semantically correct, I am not sure if this change wouldn't make the caching useless.

Reply:
for me it was needed to prevent using cache after changing configuration for gcc